Src/Sound/PGESound.cpp

```diff
@@ -0,0 +1,31 @@
+// I couldn't get OpenAL to link to both the engine and SCPCB at the same time.
```

Comment:
You don't link libraries several times, plain and simple.
We're not merging any OpenAL implementation.

Reply:
This is leftovers and is no longer relevant. Basically, there needs to be a sound system in PGE.